examples/specialty_plots/ishikawa.py

```diff
@@ -0,0 +1,228 @@
+"""
+===============
+Ishikawa Diagrams
+===============
+
+Ishikawa Diagrams are useful for visualizing the effect to many cause relationships
```

Comment:
Wikipedia [says](https://en.wikipedia.org/wiki/Ishikawa_diagram) that these are also known as "fishbone diagrams, herringbone diagrams, and cause-and-effect diagrams". Could you add those alternate names to the description so they show up in a search?